packages/client/src/index.ts

```diff
@@ -30,14 +30,12 @@ export const getPenumbraManifest = async (
   return manifestJson;
 };
 
-export const getAllPenumbraManifests = (): Record<
-  keyof (typeof window)[typeof PenumbraSymbol],
-  Promise<PenumbraManifest>
-> =>
+/** Fetch all manifests for all providers available on the page. */
+export const getAllPenumbraManifests = (signal?: AbortSignal) =>
```

Comment:
suggestion: add function return types. When working on Nextjs example, my IDE had troubles collecting the types correctly, so the explicit typing might help